### PR TITLE
Add SMB to zgrab_schema

### PR DESCRIPTION
**Pull request by ajholland**

### CODE REVIEW

File: zgrab_schema.py [hunk 1]
```diff
@@ -670,6 +670,16 @@
 
 zschema.registry.register_schema("zgrab-s7", zgrab_s7)
 
+zgrab_smb = Record({
+    "data":SubRecord({
+        "smb":SubRecord({
+           "smbv1":Boolean(),
```
Comment (zakird) [inline zgrab_schema.py]: i might label that last field `smb1_support` to make it clear that it's not a subrecord.

**Comment by zakird** [comment]
This looks fine. You probably also want to fix the code to match this ;)

**Comment by dadrian** [comment]
@cdzombak I believe @ajholland is out this week, can you take over this PR?

**Comment by cdzombak** [comment]
> can you take over this PR?

@dadrian yes. I am not familiar with the goal for this work, though, and I see no open issues on this repo or in Trello. What do you need me to do?

**Comment by ajholland** [comment]
Since this is a change to the schema and most? everything else was already merged I think this just needs to be merged

**Comment by dadrian** [comment]
@cdzombak Zakir's comment still needs to be addressed, as far as I can tell. I may be mistaken?

> This looks fine. You probably also want to fix the code to match this ;)

If I"m understanding correctly, @zakird requested a name in the output be changed, which @ajholland updated in the schema, but the necessary change to the actual JSON ouput in the SMB code needs to be made as well (e.g. changing the `json:blah` struct tag).

**Comment by cdzombak** [comment]
@dadrian makes sense. I'll get to that tomorrow.

**Comment by cdzombak** [comment]
1f3067b makes the necessary name change in the grabber itself.

On my home network, a scan results in the following JSON, which seems to match the schema:

```json
{"ip":"192.168.0.165","timestamp":"2017-06-20T09:38:34-04:00","data":{"smb":{"smbv1_support":false}},"error":"Invalid NetBIOS Session message","error_component":"smb"}
```

**Pull request merged**